app/controllers/documents/overview_controller.rb

```diff
@@ -15,7 +15,10 @@ def section_title
     end
 
     def recommended_document_types
-      DocumentNavigation.new(self).types_for_intake(current_intake)
+      document_types = DocumentNavigation.new(self).types_for_intake(current_intake)
+      include_requested_documents = @documents.any? { |doc| doc.document_type == "Requested" }
```

Comment:
this might a little more efficient as a query instead of an map block, for example, `@documents.where(document_type: "Requested").exists?`